models/topic-model.js

```diff
@@ -43,11 +43,16 @@ ORDER BY
   })
 }
 
-const fetchCommentsByArticle = (article_id) =>{
-  return db.query("SELECT * FROM COMMENTS WHERE article_id = $1 ORDER BY created_at DESC;", [article_id]).then(({rows})=>{
-    return rows;
-  })
-}
+const fetchCommentsByArticle = async (article_id) => {
```

Comment:
It's good to stay consistent, either with ```async/await``` or ```.then```  throughout, so I'd encourage you to keep one or the other for the sake of consistency in your portfolio piece :)